src/main/java/cms/gov/madie/measure/resources/BundleController.java

```diff
@@ -30,18 +31,29 @@ public class BundleController {
   public ResponseEntity<String> getMeasureBundle(
       @PathVariable String measureId,
       Principal principal,
-      @RequestHeader("Authorization") String accessToken) {
+      @RequestHeader("Authorization") String accessToken,
+      @RequestParam(required = false, defaultValue = "calculation", name = "bundleType")
+          String bundleType) {
     Optional<Measure> measureOptional = measureRepository.findById(measureId);
-    log.info(
-        "User [{}] is attempting to create a new measure bundle for [{}]",
-        principal.getName(),
-        measureId);
+
+    if ("calculation".equalsIgnoreCase(bundleType)) {
+      log.info(
+          "User [{}] is attempting to create a new measure bundle for [{}]",
+          principal.getName(),
+          measureId);
+    } else {
+      log.info("User [{}] is attempting to export measure [{}]", principal.getName(), measureId);
+    }
```

Comment:
this seems to be a duplicate check. can be moved to the code block at lines 53-57

Reply:
@adongare I agree with you, but the card said one method, and based on passed in value, we determine if it's calculation or export bundles. Maybe we can have a small meeting to determine what should be the best?

Reply:
I think it makes sense to meet and talk. looks like there is some confusion here. `getMeasureBundle` should give you measure bundle based on bundle type. However, `getMeasureBundleForExport`(which ideally should be `getMeasureExport` or `exportMeasure`) should not return bundle. Instead, it should return export(zip containing bundle json and xml, library cql and library resources, human readable) and does not need to have parameter `bundleType`. So, they are two different functionalities.

Reply:
@adongare Ok thanks so much!